### ISSUE TITLE
question about normalization in furnace/utils/img_utils.py

### ISSUE THREAD

**Issue by minzhouGithub** [ycszen/TorchSeg#39]
I tried to visualize the training data to verify whether my data is processed correctly. And I found that you didn't divide the input data by std (furnace/utils/img_utils.py). Why??

**Comment by ycszen** [comment]
I have fixed the normalize function in the new version.
